conf/checksums

```diff
@@ -28,6 +28,8 @@ spark:2.3.4:0241cbe73d3ad4fec1c655d839facc16e647ff9a3825f51c78d9fb270a753da4
 spark:2.3.3:a5c4a54db5163df25110534f36e13d6ae65758d45d1bc8bfcd7e27304e716a23
 spark:2.3.2:3387107155d62f04ccf6bcaf2e00a69a0de5ae5df875348d93147743c206f0a8
 spark:2.2.2:023b2fea378b3dd0fee2d5d1de6bfaf2d8349aefe7be97a9cbcf03bbacc428d7
+zookeeper:3.5.6:db24700e453f2ad4b6b789d553fe828ccceef3977a7e6b36389580ec92397a7e
+zookeeper:3.5.5:c5ff531cbda56c157199ab80632dc50ffefa8b7cbe866a0431345d3c4d72bbd1
```

Comment:
Adding 3.5.5 here seems like it's encouraging the use of a version that has known bugs already fixed in 3.5.6, which people should use instead. Is there a good reason to add 3.5.5 here, or is it just being added for "completeness"? If the latter, I would leave it out... the only reason we have so many listed already is because they built up over time. We should probably be pruning this list, especially if the listed versions have been aged out of the ASF mirrors system.